models/pydantic/models.py

```diff
@@ -1,11 +1,16 @@
-from datetime import date
-from pydantic import BaseModel, ConfigDict
+from datetime import date, datetime
+from pydantic import BaseModel, ConfigDict, computed_field
 
 
 class AnimalCreate(BaseModel):
     animal_type: str
     name: str
     birth_date: date
+    animal_breed: str
+    photo: str
+    @computed_field
+    def age(self) -> int:
+        return datetime.now().year - self.birth_date.year
```

Comment:
In case when birthday is in December 2022 and now October 2023, it will calculate like full year.
But in our case it's enough